src/types/Posts.ts

```diff
@@ -0,0 +1,11 @@
+import { Comment } from './Comment';
+import { User } from './User';
+
+export interface Posts {
+  userId: number;
+  id: number;
+  title: string;
+  body: string;
+  comments?: Comment[] | undefined;
```

Comment:
```suggestion
  comments?: Comment[];
```
You already have optional operator - "?"